includes/util.php

```diff
@@ -495,16 +495,78 @@ function newspack_get_countries() {
 	return $countries_options;
 }
 
+/**
+ * Pick either white or black, whatever has sufficient contrast with the color being passed to it.
+ * (Copied from the Newspack theme: https://github.com/Automattic/newspack-theme/blob/6dc4e89a65c465abdd207d990e313921f2972a9a/newspack-theme/inc/template-functions.php#L547)
+ *
+ * @param  string $hex Hexidecimal value of the color to adjust.
+ *
+ * @return string Either black or white hexidecimal values.
+ *
+ * @ref https://stackoverflow.com/questions/1331591/given-a-background-color-black-or-white-text
+ */
+function newspack_get_color_contrast( $hex ) {
```

Comment:
This utility function lives in newspack theme, but given not every publisher will be using this theme, and given how useful this function is, I've copied it over to the main newspack plugin. 

I'm happy to move it back if someone feels it shouldn't be copied over though!